test/parallel/parallel.status

```diff
@@ -14,6 +14,11 @@ test-crypto-keygen: PASS,FLAKY
 # https://github.com/nodejs/node/issues/41201
 test-fs-rmdir-recursive: PASS, FLAKY
 
+# Windows on ARM
+[$system==win32 && $arch==arm64]
+test-child-process-exec-cwd: SKIP
+test-permission-deny-fs-wildcard: SKIP
```

Comment:
https://github.com/nodejs/node/pull/47095

Reply:
Hey, @richardlau is there something else I should change for this to land? Please note that this PR aims to fix JS test suites on Windows on ARM. That's why native suites failed in the CI build I provided before.

Reply:
I plan to fix this test, but until I can do that I'd like to move forward with https://github.com/nodejs/build/issues/3046 so we can prevent any ARM64 regressions. @targos do you have strong feelings against landing this for now?